src/main/resources/db/changelog/changes/create-accommodation-table.yaml

```diff
@@ -41,7 +41,7 @@ databaseChangeLog:
                   constraints:
                     nullable: false
               - column:
-                  name: is_deleted
+                  name: deleted
```

Comment:
it was ok by the way ;)

Reply:
Yeah I changed because of multiple warnings during mvn clean package, anyway, I changed my mapper logic so this changed could be reverted, but I don`t think it`s necessary